src/app/SubmissionList.jsx

```diff
@@ -149,6 +192,30 @@ const styles = {
       marginBottom: 10,
       boxShadow: '0px 0px 5px 0px rgba(0,0,0,0.2)'
     },
+    pagination: {
```

Comment:
I updated the code so that the pagination bar is positioned absolutely to the bottom of the window